src/MusicStore/Controllers/AccountController.cs

```diff
@@ -65,6 +71,7 @@ public IActionResult Login(string returnUrl = null)
             }
             else
             {
+                _logger.LogInformation("Failed to login {userName}.", model.Email);
```

Comment:
I agree with @kichalla , from a business perspective this is a warning - it's something that could raise a red flag during an audit of logs.